src/ray/core_worker/core_worker.cc

```diff
@@ -2119,8 +2132,16 @@ Status CoreWorker::AllocateReturnObject(const ObjectID &object_id,
   }
   // Leave the return object as a nullptr if the object already exists.
   if (!object_already_exists) {
+    std::vector<rpc::ObjectReference> contained_refs;
```

Comment:
is it possible to reuse `GetObjectRefs` we introduced earlier in this file?

Reply:
Oh yes, good point!